### ISSUE TITLE
There is a problem with how Rake tests the #move method.

### ISSUE THREAD

**Issue by Sdcrouse** [learn-co-curriculum/tic-tac-toe-rb#67]
Hello. I am reopening an issue that I raised in July, as it does not appear to have been fixed. However, as I do not know how Rake or Rakefiles work, I am not sure if it CAN be fixed.

The issue that I am referring to is #64, which I raised here: https://github.com/learn-co-curriculum/tic-tac-toe-rb/issues/64

To summarize the issue, there is a name collision whenever you test the #move method with Rake. I have attached a screenshot to add some clarity. Note that I wrote the #move method correctly, since it passed the Rspec tests.

Thanks for looking into this again!

Sdcrouse

![ruby tic tac toe screenshot](https://user-images.githubusercontent.com/38870640/45601429-7fdfcc00-b9c1-11e8-99f8-b73a5d3e4f18.gif)

**Comment by maxwellbenton** [comment]
Hi @Sdcrouse,

Thank you for bringing this to our attention and providing details on the issue! I was able to recreate the problem using the image you provided. To avoid conflict with Rake, I've changed the "move" method to "make_move" in the readme, tests and solution.

Thank you for this as well as all of your work notifying us of issues on our lessons, @Sdcrouse. The curriculum team at Flatiron School greatly appreciates the time you've taken!
